content/blog/posts/texas-attorney-general’s-sites-for-death-and-shooting-reports-go-dark.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Texas Attorney General’s Sites for Death and Shooting Reports Go Dark "
+subtitle: ""
+date: 2021-03-20
+authors:
+  - Eva Ruth Moravec
+---
+For five weeks, the Texas Office of the Attorney General’s websites that house reports on [officer-involved shootings](https://oagtx.force.com/oisreports/apex/OISReportsPage) and [deaths in custody](https://oagtx.force.com/cdr/cdrreportdeaths) have been malfunctioning, restricting access to records that detail important interactions between civilians and law enforcement. 
```

Comment:
```suggestion
For five weeks, the Texas Office of the Attorney General’s websites that house reports on [officer-involved shootings](https://oagtx.force.com/oisreports/apex/OISReportsPage) and [deaths in custody](https://oagtx.force.com/cdr/cdrreportdeaths) have been malfunctioning, cutting off access to records that detail important (and often lethal) interactions between civilians and law enforcement. 
```

Reply:
I think a stronger word than restricting is needed because to me that suggests that some people still have access to the info when in reality no one (outside of the agency) does

Reply:
thanks - great suggestion!